hedera-node/src/main/java/com/hedera/services/contracts/sources/EvmSigsVerifier.java

```diff
@@ -117,6 +117,31 @@ boolean hasActiveSupplyKey(
             Address activeContract,
             WorldLedgers worldLedgers);
 
+    /**
+     * Determines if the target token has an active admin key given the cryptographic signatures
+     * from the {@link com.hederahashgraph.api.proto.java.SignatureMap} that could be verified
+     * asynchronously; plus the given recipient and contract of the current {@link
+     * org.hyperledger.besu.evm.frame.MessageFrame}.
+     *
+     * <p>If the admin key includes a {@code contractID} key matching the contract address, or a
+     * {@code delegatableContractId} key matching the recipient address, then those keys must be
+     * treated as active for the purposes of this test.
```

Comment:
Hey, I can see all keys' descriptions contain the "test" paragraph, not sure if it's because they are passed as "KeyActivationTest" in KeyActivationUtils.validateKey?